### ISSUE TITLE
Decide whether to teach subset() function

### ISSUE THREAD

**Issue by cyouh95** [anyone-can-cook/rclass1#15]
There are 3 ways to subset:

- Use brackets
- Tidyverse `filter()`
- BaseR `subset()`

Is there anything that can be done w/ `subset()` that can't be done using brackets?

**Comment by cyouh95** [comment]
I think brackets can do everything `subset()` can do - it just might be easier for `subset()` to do some things, specifically excluding rows/elements whose condition is `NA`, but it's still possible to do w/ brackets by using `which()`. There was a few slides in the lecture that covered this.

The lecture originally included some slides comparing these 3 subsetting methods, then there's also a section at the end specifically for `subset()`. Since I removed all tidyverse stuff from the lec, I also removed `subset()` from those comparison slides, so everything `subset()` related should only be under the `Subsetting using subset() function` section, if we choose to skip over it. @ozanj @mpatricia01
